ivy_tests/test_ivy/test_frontends/test_jax/test_devicearray.py

```diff
@@ -510,6 +510,45 @@ def test_jax_devicearray_nonzero(
     )
 
 
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="jax.numpy.array",
+    method_name="prod",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=["int64"],
```

Comment:
I can't fully understand why this is the only supported dtype

Reply:
Hey @hirwa-nshuti , thanks for the review. I've also tried testing with other valid dtypes but the ground-truth is always returning `int64` causing the tests to fail with an `AssertionError` since returned dtype is different from ground truth.  I haven't encountered this problem before, is there a way to solve this issue?

Reply:
Then if there is a special promotion being done should be handled in the frontend function not on the testing side to ensure that we cover all the corner cases